README.md

```diff
@@ -958,27 +1196,39 @@ Specifies the database to which you are granting access.
 
 ##### `object_type`
 
-Specifies the type of object to which you are granting privileges. Valid options: `DATABASE`, `SCHEMA`, `SEQUENCE`, `ALL SEQUENCES IN SCHEMA`, `TABLE` or `ALL TABLES IN SCHEMA`.
+Specifies the type of object to which you are granting privileges.
+
+Valid options: 'DATABASE', 'SCHEMA', 'SEQUENCE', 'ALL SEQUENCES IN SCHEMA', 'TABLE' or 'ALL TABLES IN SCHEMA'.
 
 ##### `object_name`
 
 Specifies name of `object_type` to which to grant access.
 
 ##### `port`
 
-Port to use when connecting. Default: undef, which generally defaults to port 5432 depending on your PostgreSQL packaging.
+Port to use when connecting.
+
+Default value: `undef`, which generally defaults to port 5432 depending on your PostgreSQL packaging.
 
 ##### `privilege`
 
-Specifies the privilege to grant. Valid options: `ALL`, `ALL PRIVILEGES` or `object_type` dependent string.
+Specifies the privilege to grant.
+
+Valid options: 'ALL', 'ALL PRIVILEGES' or 'object_type' dependent string.
 
 ##### `psql_db`
 
-Specifies the database to execute the grant against. _This should not ordinarily be changed from the default_, which is `postgres`.
+Specifies the database to execute the grant against.
+
+_This should not ordinarily be changed from the default_, 
```

Comment:
Please put this in **bold** instead of _italics_ for consistency with the previous warning like this.